chapters/en/module3.md

```diff
@@ -12,19 +12,708 @@ id: 3
 <slides source="module3/module3_00" shot="0" start="3:5707" end="4:5306"> </slides>
 </exercise>
 
-<exercise id="1" title="How to visualize data from a single column" type="slides,video">
+<exercise id="1" title="How To Visualize Data From a Single Column" type="slides,video">
 <slides source="module3/module3_01" shot="1" start="0:003" end="07:12"> </slides>
 </exercise>
 
-<exercise id="2" title="Visulize distributions with density plots" type="slides,video">
-<slides source="module3/module3_02" shot="1" start="0:003" end="07:12"> </slides>
+
+<exercise id="2" title="True or False: Distributions">
+
+**True or False**       
+*When we are visualization data from a single column, we are (usually) more interested in the shape/distribution in general, than individual points.*
+
+
+<choice id="1" >
+
+<opt text="True"  correct="true">
+
+You got it!  
+
+</opt>
+
+
+<opt text="False">
+
+We are usually interested in the values as a collective.
+
+</opt>
+
+
+</choice>
+
+**True or False**       
+*Layered histograms are more interpretable than stacked histograms.*
+
+<choice id="2" >
+
+<opt text="True"  >
+
+This isn't always true. In fact, depending on the question you are attempting to answer, layered histograms can do the opposite.
+
+</opt>
+
+<opt text="False" correct="true">
+
+You are on a roll!
+
+</opt>
+
+</choice>
+
+</exercise>
+
+
+<exercise id="3" title="Multple Choice Questions on Single Column Plots">
+
+**Question 1**      
+ Why can rugplots be problematic?
```

Comment:
You got it!

Reply:
I am generally a bit split on how much to rely on these names. They are useful shortcuts, but it is also nice to teach students to think of the content and components of the plots rather than just recall a name by heart. In either case, an image will solve most of this, but you might want to add something more general e.g.

"When is it problematic to visualize all observations in a dataset, such as in this rugplot?"